### DISCUSSION TITLE
npm publish --dry-run should check for duplicate package version

### DISCUSSION THREAD

**Discussion by navbruce** [npm/feedback#404] [CLI] [answered]
Currently, `npm publish --dry-run` will report success where `npm publish` will fail with the error:

```
You cannot publish over the previously published versions: 1.2.3.
```

This makes `npm publish --dry-run` not as useful as it could be for CI workflow.

Is there something I'm missing here?

**Comment by MylesBorins** [comment, marked as answer]
@navbruce this seems like something that should likely be posted as an issue to the https://github.com/npm/cli repo as a bug report
